drape/gl_functions.cpp

```diff
@@ -255,9 +255,22 @@ void GLFunctions::Init(dp::ApiVersion apiVersion)
     glUnmapBufferFn = &::glUnmapBuffer;
 
 #elif defined(OMIM_OS_LINUX)
+#if defined(OMIM_OS_LINUX_HAS_EGL)
 
-    CHECK(false, ("OpenGLES2 is not yet supported"));
+    glGenVertexArraysFn = (TglGenVertexArraysFn)eglGetProcAddress("glGenVertexArraysOES");
+    glBindVertexArrayFn = (TglBindVertexArrayFn)eglGetProcAddress("glBindVertexArrayOES");
+    glDeleteVertexArrayFn = (TglDeleteVertexArrayFn)eglGetProcAddress("glDeleteVertexArraysOES");
+    glMapBufferFn = (TglMapBufferFn)eglGetProcAddress("glMapBufferOES");
+    glUnmapBufferFn = (TglUnmapBufferFn)eglGetProcAddress("glUnmapBufferOES");
+    glMapBufferRangeFn = (TglMapBufferRangeFn)eglGetProcAddress("glMapBufferRangeEXT");
+    glFlushMappedBufferRangeFn =
+        (TglFlushMappedBufferRangeFn)eglGetProcAddress("glFlushMappedBufferRangeEXT");
```

Comment:
```suggestion
    glFlushMappedBufferRangeFn = (TglFlushMappedBufferRangeFn)eglGetProcAddress("glFlushMappedBufferRangeEXT");
```

Reply:
This code has changed regardless, please check again.